common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -508,3 +508,65 @@ def test_studio_settings_page_a11y(self):
         })
 
         self.settings_page.a11y_audit.check_for_accessibility_errors()
+
+
+@attr('a11y')
+class StudioSubsectionSettingsA11yTest(StudioCourseTest):
+    """
+    Class to test accessibility on the subsection settings modals.
+    """
+
+    def setUp(self):
+        browser = os.environ.get('SELENIUM_BROWSER', 'firefox')
+
+        """
+        This test will fail if run using phantomjs < 2.0, due to an issue with bind()
+        See https://github.com/ariya/phantomjs/issues/10522 for details.
+
+        The course_outline uses this function, and as such will not fully load when run
+        under phantomjs 1.9.8. So, to prevent this test from timing out at course_outline.visit(),
+        force the use of firefox vs the standard a11y test usage of phantomjs 1.9.8.
+
+        This block can be removed once https://openedx.atlassian.net/browse/TE-1047 is resolved.
+        """
+        if browser == 'phantomjs':
+            browser = 'firefox'
+
+        with patch.dict(os.environ, {'SELENIUM_BROWSER': browser}):
+            super(StudioSubsectionSettingsA11yTest, self).setUp(is_staff=True)
```

Comment:
Makes sense to me. We're doing a workaround and have logged a ticket pointing out that we shouldn't have to. I also wonder if we should pivot to installing phantomjs via npm, but that itself would be another story.